website_snippet_carousel_product/static/src/js/s_product_carousel_frontend.js

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Tecnativa - Alexandre Díaz
+// License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+odoo.define("website_snippet_carousel_product.s_product_carousel", function(require) {
+    "use strict";
+
+    const core = require("web.core");
+    const sAnimation = require("website.content.snippets.animation");
+
+    const _t = core._t;
+
+    sAnimation.registry.js_product_carousel = sAnimation.Class.extend({
+        selector: ".js_product_carousel",
+        disabledInEditableMode: false,
+
+        /**
+         * @override
+         */
+        start: function() {
+            const _this = this;
+            const limit = Number(this.$target.attr("data-products-limit")) || 12;
+            const domain = this.$target.attr("data-domain") || "[]";
+            const products_per_slide =
+                Number(this.$target.attr("data-products-per-slide")) || 4;
+            let interval = Number(this.$target.attr("data-interval"));
+            if (_.isNaN(interval)) {
+                interval = 5000;
+            }
+
+            // Prevent user edition
+            this.$target.attr("contenteditable", "False");
```

Comment:
The values on the documentation are in lowercase...
```suggestion
            this.$target.attr("contenteditable", "false");
```
But maybe you can use "o_not_editable" css class directly in the snippet element.